package.json

```diff
@@ -62,7 +62,7 @@
     "setenv": "echo REACT_APP_VERSION=$(git rev-parse HEAD) REACT_APP_BUILD_TIMESTAMP=$(date -u \"+%s000\") EXTEND_ESLINT=true",
     "start": "env $(yarn -s setenv) react-app-rewired start",
     "build": "env $(yarn -s setenv) react-app-rewired build",
-    "postbuild": "rm build/config.json",
+    "postbuild": "rm build/config.json && echo && echo '*** Be sure to copy a config/xxx.json to build/config.json. ***'",
```

Comment:
[Asking for my edification] How does  ^^^  work?

Reply:
It's just a message that appears after you run `yarn build`. It'll be sitting pretty close to your prompt in the terminal and might be enough to prevent a mistake.